assets/src/dashboard/components/cardGridItem/cardTitle.js

```diff
@@ -157,7 +157,10 @@ const CardTitle = ({
 CardTitle.propTypes = {
   title: PropTypes.string.isRequired,
   secondaryTitle: PropTypes.string,
-  status: PropTypes.oneOf(Object.values(STORY_STATUS)),
+  status: PropTypes.oneOf([
+    ...Object.values(STORY_STATUS),
+    ...Object.values(TEMPLATES_GALLERY_STATUS),
+  ]),
```

Comment:
Super minor nit, but we might want to create a PropType out of this.

Reply:
that's a great idea!